spec/models/metric/statistic_spec.rb

```diff
@@ -5,7 +5,7 @@
                          :zone => Zone.first)
     end
 
-    hour = Time.parse(Metric::Helper.nearest_hourly_timestamp(Time.now)).utc
+    hour = Time.now.utc.change(:min => 0)
```

Comment:
Did not know about this method...can we just kill the "nearest_hourly_timestamp" method entirely and just do this?